agents-api/pyproject.toml

```diff
@@ -66,9 +67,12 @@ datamodel-codegen \
   --input-file-type openapi \
   --output agents_api/autogen/ \
   --output-model-type pydantic_v2.BaseModel \
+  --strict-types bool \
   --strict-nullable \
   --allow-population-by-field-name \
+  --allow-extra-fields \
```

Comment:
Actually I think we should turn off allow extra. It might cause more problems than it solves